store-and-retrieve-data/use-storagehub-sdk/get-started.md

```diff
@@ -78,6 +78,139 @@ Add the core and MSP client packages to your project. These libraries provide th
     npm install @storagehub-sdk/core @storagehub-sdk/msp-client
     ```
 
+## Initialize the StorageHub SDK
+
+In this step, you will learn how to initialize all the clients you will need while building with the StorageHub SDK. You will go through installing extra dependencies, setting up `walletClient`, `publicClient`, `storageHubClient`, `polkadotApi`, `mspClient`, and a few helper functions related to the `mspClient` to help get you started with things like checking MSP health and user authentication.
```

Comment:
```suggestion
Follow the steps in this section to set up the clients needed to work with the StorageHub SDK, allowing you to interact with DataHaven and manage your data.
```

Let's not get into all of the client names yet because if a user is following this guide, they're just starting out and none of this means anything to them yet anyways.